manifests/plugin/remote_execution/ssh.pp

```diff
@@ -0,0 +1,67 @@
+# = Foreman Proxy Remote Execution SSH plugin
```

Comment:
I'm not sure if this is the right place or that it should be named ```remote_execution_ssh```. Do we expect any other ```remote_execution_*``` plugins?